packages/main/src/themes/Tag.css

```diff
@@ -82,15 +82,20 @@
 ::slotted([ui5-icon]) {
 	width: var(--_ui5-tag-icon-width);
 	min-width: var(--_ui5-tag-icon-width);
-	height: var(--_ui5-tag-height);
-	min-height: var(--_ui5-tag-height);
+	height: var(--_ui5-tag-icon-height);
 	color: inherit;
 	pointer-events: none;
+	align-self: flex-start;
 }
 
-:host([wrapping-type="Normal"]) [ui5-icon],
-:host([wrapping-type="Normal"]) ::slotted([ui5-icon]) {
-	align-self: flex-start;
+:host([_is-tag-design]) [ui5-icon],
```

Comment:
_is-tag-design is regarding design="Set3", as it is deleted with this pull https://github.com/SAP/ui5-webcomponents/pull/9115/files#diff-9dc0bd3858a768f765383f49db1243e566677dae75941c9e40048637c57e0b7d

You can delete this selector regarding"_is-tag-design"